storage_manager_new.py

```diff
@@ -54,135 +58,122 @@ def show_all_items(self):
         print('\n')
 
     def show_items_from_category(self):
-        category = Interface.get_user_choice(
-            'Print items from category:').lower()
-        for item in Item.instances:
+        category = input(
+            'Print items from category:\n').lower()
+        print('')
+        for item in self.items:
             if category == item.category.lower():
                 print(f'({item.category})  {item.type} {item.model}')
         print('\n')
 
     def delete_item(self):
-        File = open(Storage.database_file, 'r')
+        File = open(self.database_file, 'r')
         items_in_file = File.readlines()
         File.close()
-        choice = Interface.get_user_choice(
-            'Which item would you like to delete? Please provide specific info about type or model.')
+        item_to_delete = input(
+            'Which item would you like to delete? Please provide specific info about type or model.\n')
         for item in items_in_file:
-            if choice.lower() in item.lower():
+            if item_to_delete.lower() in item.lower():
                 confirmation = None
                 while confirmation != 'y' and confirmation != 'n':
```

Comment:
Or `while confirmation not in ('y', 'n'):` or even `while not confirmation` as any non-empty string is truthy in Python - just showing another ways, yours is good as well.

Reply:
I think I'll change this part to `while confirmation not in ('y', 'n', 'menu')`, since there is also a condition for using string `'menu'` as `confirmartion`, and this way it will be more readable than adding another `and confirmation != 'menu'`.